tcrdist/repertoire.py

```diff
@@ -1338,27 +1333,75 @@ def _deduplicate(cell_df, index_cols):
     clones = cell_df.groupby(index_cols)['count'].agg(np.sum).reset_index()
     return clones
 
-def _compute_pairwise(sequences, metric = "nw", processes = 2, user_function = None, **kwargs):
-    """
-    Wrapper for pairwise.apply_pw_distance_metric_w_multiprocessing()
+def _compute_pairwise(sequences, metric='nw', processes=2, user_function=None, **kwargs):
+    """Wrapper for pairwise.apply_pw_distance_metric_w_multiprocessing()
 
     Parameters
     ----------
     sequences : list
-
     metric : string
-
     processes : int
-
     user_function : function
+    **kwargs : keyword arguments passed to metric function
 
     Returns
     -------
-
     pw_full_np : np.ndarray
-        matrix of pairwise comparisons
-
+        matrix of pairwise comparisons"""
+    # processes = 1
+    
+    if metric == 'nw':
+        metric_func = pwseqdist.metrics.nw_metric
+    elif metric == 'hamming':
+        metric_func = pwseqdist.metrics.nw_hamming_metric
+    elif metric == 'tcrdist_cdr3':
+        metric_func = pairwise.tcrdist_cdr3_metric
+    elif metric in ['tcrdist_cdr1', 'tcrdist_cdr2', 'tcrdist_cdr2.5', 'tcrdist_pmhc']:
+        metric_func = pairwise.tcrdist_cdr1_metric
+    elif metric == 'hamming_aligned':
+        metric_func = pwseqdist.metrics.hamming_distance
+    elif not user_function is None:
+        metric_func = user_function
+    else:
+        msg = 'repertoire._compute_pairwise: metric %s is not supported'
+        raise ValueError(msg % metric)
+    
+    if metric in ['nw', 'hamming']:
+        if not 'matrix' in kwargs:
+            kwargs['matrix'] = 'blosum62'
+
+    dvec = pwseqdist.apply_pairwise_sq(sequences.values, metric_func,
+                                  ncpus=processes, **kwargs)
```

Comment:
CRUCIAL CHANGES: LINE 1373-1374 -  in tcrdist.repertoire.TCRrep._compute_pairwise()  is where new pswseqdist is used